dandi/models.py

```diff
@@ -113,6 +113,185 @@ def diff_models(model1, model2):
             print(f"{field} is different")
 
 
+DATACITE_CONTRTYPE = [
```

Comment:
As this is only used for checking containment, it may be more efficient to make it a set.

Reply:
yes, can move it to set